### PR TITLE
RUN-3874 flush cookies

### PR DESCRIPTION
**Pull request by wenjunche**
[W10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5a9570ab6a994a57faa5c984)
[W7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5a956fd36a994a57faa5c983)

[New Test](https://testing-dashboard.openfin.co/#/app/tests/5a956ecb6a994a57faa5c982/edit)

[Adapter](https://github.com/openfin/javascript-adapter/pull/399)

### CODE REVIEW

File: src/browser/api/system.js [hunk 1]
```diff
@@ -627,6 +627,11 @@ exports.System = {
             errorCallback(`Error getting cookies`);
         }
     },
+    flushCookieStore: function(callback) {
+        session.defaultSession.cookies.flushStore(() => {
+            callback();
+        });
```
Comment (whyn07m3) [inline src/browser/api/system.js]: Suggestion:
```
session.defaultSession.cookies.flushStore(callback);
```

File: src/browser/api_protocol/api_handlers/system.js [hunk 2]
```diff
@@ -452,6 +453,12 @@ function SystemApiHandler() {
 
     }
 
+    function flushCookieStore(identity, message, ack, nack) {
+        System.flushCookieStore(function() {
+            ack(successAck);
+        });
```
Comment (whyn07m3) [inline src/browser/api_protocol/api_handlers/system.js]: Suggestion:
```
System.flushCookieStore(() => ack(successAck));
```

**Comment by rdepena** [comment]
Is there a way to flush it for a given Application (Render process or Domain) instead of across the board ?

**Comment by rdepena** [comment]
Wenjun talked some sense into me.

**Pull request merged**